### PR TITLE
Add `slc` subcommand to seek line to free the cache and added `sl?` help.

### PR DESCRIPTION
**Pull request by zlowram**
Related to issue #3866.

### CODE REVIEW

File: libr/core/cmd_seek.c [hunk 1]
```diff
@@ -341,6 +341,15 @@ static int cmd_seek(void *data, const char *input) {
 				off = core->lines_cache[curr+sl_arg];
 				r_core_seek (core, off, 1);
 				break;
+			case 'c':
+				R_FREE (core->lines_cache);
+				break;
+			case '?':
+				eprintf ("Usage: sl+ or sl- or slc\n");
+				eprintf (" sl [line]    - Seek to absolute line\n");
```
Comment (radare) [inline libr/core/cmd_seek.c]: Use a single eprintf statement

File: libr/core/cmd_seek.c [hunk 2]
```diff
@@ -341,6 +341,15 @@ static int cmd_seek(void *data, const char *input) {
 				off = core->lines_cache[curr+sl_arg];
 				r_core_seek (core, off, 1);
 				break;
+			case 'c':
+				R_FREE (core->lines_cache);
+				break;
+			case '?':
+				eprintf ("Usage: sl+ or sl- or slc\n");
```
Comment (radare) [inline libr/core/cmd_seek.c]: Use r_core_help api to get full colors and free indentation

File: libr/core/cmd_seek.c [hunk 3]
```diff
@@ -341,6 +341,15 @@ static int cmd_seek(void *data, const char *input) {
 				off = core->lines_cache[curr+sl_arg];
 				r_core_seek (core, off, 1);
 				break;
+			case 'c':
+				R_FREE (core->lines_cache);
```
Comment (radare) [inline libr/core/cmd_seek.c]: What about cachesz=-1 too? Put this in a separe function. Maybe move all the line stuff into core/line.c?

**Comment by zlowram** [comment]
This commit is on #3872 too, closing this one.

**Pull request closed without merging**